### ISSUE TITLE
General formatting and organization clean-up

### ISSUE THREAD

**Issue by aliciacanipe** [spacetelescope/jwst_validation_notebooks#92]
Since this is a low-priority issue, I'll file a ticket for minor formatting things to fix in terms of making the folder and notebook names consistent. 

**Folder** names should either be the name of a step (e.g., source_catalog) or the name of a pipeline (e.g., calwebb_detector1, calwebb_tso3). **Notebook** names should be jwst_<step/pipeline>_<instrument/mode>.ipynb, where <pipeline> can just be the shorter suffix (detector1, tso3, image2, etc). Does that seem reasonable @cracraft and @york-stsci? 

Following those suggestions:

1. We should see if it's possible to move the notebooks in the “regression_tests” folder into the individual step name folders, e.g., the notebook that's in the regression_test —> tweakreg folder can just go in the normal tweakreg folder; however, we should leave the jwst_instrument_regression_tests.ipynb notebooks in the regression_tests folder.
2. The “image3” folder should be named calwebb_image3
4. The calwebb_coron3 notebooks should be named jwst_coron3_...ipynb to be consistent with the other notebooks
5. The caldetector1-miri-lrs-tso.ipynb notebook should be renamed to jwst_detector1_miri_lrstso.ipynb

**Comment by york-stsci** [comment]
I would prefer if the regression tests were kept in their own directory. They can often take a long time to run, and when testing out the convert.py script you can specify the directory with the --notebook-path command-line option but it's substantially more difficult to not run a single specific notebook, and I would prefer, if possible, that it be generally possible to test all the notebooks in a directory without worrying that one of them will take a long time to run.

**Comment by aliciacanipe** [comment]
True. Maybe a possibility would be to adopt a system similar to how the pipeline repo is organized? We could have a folder for the regression tests, e.g.,: 
`regression_tests/jwst_<instrument>_regression_test.ipynb`

Then within each step/pipeline folder, we could have a unit tests folder, which could (maybe?) be ignored this way, e.g.,: 
`assign_wcs/unit_tests/jwst_assign_wcs_unit_tests.ipynb`
`calwebb_detector1/unit_tests/jwst_detector1_unit_tests.ipynb`

so to ignore them:
`--ignore={unit_tests} --ignore={regression_tests}`

Would that work? Since the regression tests are instrument-specific and not step- or pipeline-specific right now, it's fine for them to be separate. But in terms of generating a final report using the website output with the tests for a step or pipeline, it would be better to have the unit tests together with the other tests. We would only be generating a full report ~quarterly, so in most cases could ignore the unit and regression test notebooks. Thoughts? Just throwing out ideas.

**Comment by york-stsci** [comment]
I think that would work. In convert.py (because it inherits from the nbpages module) the command is "--exclude" rather than "--ignore", but hopefully that's not going to be a huge problem.
